ros/wavemap_ros/launch/online/pico_monstar.launch

```diff
@@ -83,6 +83,6 @@
   <include file="$(find wavemap_ros)/launch/wavemap_server.launch"
            pass_all_args="true"/>
 
-<!--  <node pkg="rosbag" type="record" name="rosbag_record" output="screen"-->
-<!--        args="/wavemap/depth_cam_input /laserMapping/imu_forward_prop -O /home/victor/catkin_ws/lidar_align_pico_monstar"/>-->
+  <node if="false" pkg="rosbag" type="record" name="rosbag_record" output="screen"
+        args="/livox/lidar /livox/imu /pico_monstar/image_rect -o /home/victor/catkin_ws/livox_monstar"/>
```

Comment:
Could this absolute user-specific path be changed to something more general based on the current WS or similar?